subsys/usb/class/cdc_acm.c

```diff
@@ -503,7 +503,7 @@ static int cdc_acm_init(struct device *dev)
 	}
 
 	dev->driver_api = &cdc_acm_driver_api;
-	nano_sem_init(&poll_wait_sem);
+	k_sem_init(&poll_wait_sem, 0, 1);
```

Comment:
I beleive the proper change would be
k_sem_init(&poll_wait_sem, 0, UINT_MAX);
If you look at the zephyr gerrit there is a change already for this.